api/client/service.go

```diff
@@ -319,8 +319,10 @@ func (cli *DockerCli) CmdServiceRolling_update(args ...string) error {
 	}
 
 	ctx := context.Background()
-	if err := cli.pullImage(ctx, *flImage); err != nil {
-		return err
+	if _, _, err := cli.client.ImageInspectWithRaw(ctx, *flImage, false); err != nil {
```

Comment:
check the error details? e.g. only pull on image not exist

Reply:
updated.